src/pages/_playground/api-explorer.tsx

```diff
@@ -21,9 +21,18 @@ export async function getStaticProps() {
     }
   }
 
-  const params = queries.getParams().addInclude(['field_cta'])
+  const params = queries
+    .getParams()
+    .addInclude([
+      'field_additional_contact',
+      'field_contact_default',
+      'field_contact_default.field_office',
+      'field_benefit_hub_contacts',
+      'field_benefit_hub_contacts.field_support_services',
+    ])
+    .addPageLimit(1)
   const data = await drupalClient.getResourceCollection(
-    'paragraph--featured_content',
+    'paragraph--contact_information',
     {
       params: params.getQueryObject(),
       withAuth: {
```

Comment:
Do we need this file committed to source control?

Reply:
i think we do want it because it's a nice helper for formatting data for `@/mocks/` (we can `console.log(json.stringify(data))` in a given query too, but i prefer this helper route